packages/container/src/components/FieldPluginContainer.tsx

```diff
@@ -311,7 +313,25 @@ export const FieldPluginContainer: FunctionComponent = () => {
         <AccordionSummary>
           <Typography variant="h3">Content</Typography>
         </AccordionSummary>
-        <AccordionDetails sx={{ position: 'relative' }}>
+        <AccordionDetails
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '2rem',
```

Comment:
Spacing should be defined with the theme, which in this case means

```suggestion
            gap: 6,
```

or equivalently:

```suggestion
            gap: (theme) => theme.spacing(6),
```

The sx prop:
- [sizing](https://mui.com/system/getting-started/the-sx-prop/#spacing)
- [callback values](https://mui.com/system/getting-started/the-sx-prop/#callback-values)